src/platform/platform_posix.c

```diff
@@ -32,12 +32,8 @@
 #include "platform_posix.c.clog.h"
 #endif
 
-#ifdef __aarch64__
-#undef CXPLAT_NUMA_AWARE
-#endif
-
 #ifdef CXPLAT_NUMA_AWARE
-#include <numa.h>
+#include <numa.h>               // If missing: `apt-get install -y libnuma-dev`
```

Comment:
I'm not sure the comment is appropriate. It represents subset of Linux distriburtiuons and not POSIX.